doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,101 @@
+============================================
+Cross Validation
+============================================
+
+Cross Validation is a type of model validation technique for figuring out how the
+results from a statistical analysis will generalize to an independent data set. It
+essentially involves training the model on a subset of the data, and testing on the
+remainder, then repeating this process for several complementary partitions and then
+combining all the validation results generated (For instance, an average) to estimate
+a performance for the final predictive model.
+
+One type of cross validation is known as :math:`k-` fold cross validation.
+
+For instance, Suppose you have the training data :math:`\mathbf{x}_1,\mathbf{x}_2, ..., \mathbf{x}_{10n}`
+for a positive integer :math:`n` , then :math:`10-` fold cross validation might for instance partition
+the data into :math:`P_1, P_2, ..., P_n` each with :math:`10` data points.
+
+Next, the model would be trained on :math:`P_1, P_2, ...,P_{n-1}` and is tested on :math:`P_n` to get accuracy
+:math:`a_1` .
+
+Then the model would train again from scratch on :math:`P_1, P_2, ...,P_{n-2}, P_{n}` and is tested on :math:`P_{n-1}`
+to get :math:`a_2`.
+
+This process is repeated until we have :math:`a_1, a_2, ..., a_n`, at which we take their average and calculate
+their standard deviation.
+
+We will perform :math:`k-` fold cross-validation with both a classification model (Linear SVM Model) and a
+regression model (Linear Ridge Regression).
+
+-------
+Example of :math:`k-` Fold Cross-Validation on a Binary Classifier
+-------
+We will use Linear SVM Model. (see :doc:`../binary/linear_support_vector_machine` for a more complete example of Linear SVM usage).
+
+Firstly, we import the training data, training labels, test data, and test labels.
+
+.. sgexample:: cross_validation.sg:create_features
+
+Next,we initialize a splitting strategy :sgclass:`CStratifiedCrossValidationSplitting`, which is needed
+to divide the dataset into :math:`k-` folds for the :math:`k-` fold cross validation (In this case, we make :math:`k=2` ).
```

Comment:
remove the "in this case", this is obvious from the code and is easier to maintain in case the code changes